app/controllers/hosts_controller.rb

```diff
@@ -907,4 +907,11 @@ def host_power_ping(result)
   def host_power_state(key)
     HOST_POWER[key].merge(:title => _(HOST_POWER[key][:title]))
   end
+
+  def host_attributes_for_templates(host)
+    host_params.select do |k,v|
+      host.class.new.attributes.include?(k) &&
```

Comment:
What is this doing? Why wo we need to create a new instance of the host class for every param?